src/ontology/mondo-edit.obo

```diff
@@ -160885,25 +160870,37 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
 id: MONDO:0008231
 name: Peyronie disease
 def: "A condition characterized by hardening of the penis due to the formation of fibrous plaques on the dorsolateral aspect of the penis, usually involving the membrane (tunica albuginea) surrounding the erectile tissue (corpus cavernosum penis). This may eventually cause a painful deformity of the shaft or constriction of the urethra, or both." [MESH:D010411]
+subset: inferred_rare
 subset: orphanet_rare {source="Orphanet:2870"}
 subset: rare
 synonym: "Induratio penis plastica" EXACT [DOID:8616, Orphanet:2870]
+synonym: "penile fibromatosis" EXACT [MONDO:0006361, NCIT:C3316]
+synonym: "penile induration" EXACT [NCIT:C3316]
+synonym: "penis fibromatosis" EXACT [MONDO:patterns/location]
 synonym: "Peyronie disease" EXACT [DOID:8616, MTH:NOCODE, OMIM:171000]
-synonym: "Peyronie's disease" EXACT [doi:10.1093/jama/9780195176339.003.0016, MONDO:LexicalVariant]
-synonym: "Peyronie's fibromatosis" RELATED [DOID:8616, NCIT:C3316]
+synonym: "Peyronie's disease" EXACT [doi:10.1093/jama/9780195176339.003.0016, MONDO:LexicalVariant, NCIT:C3316]
+synonym: "Peyronie's fibromatosis" EXACT [DOID:8616, NCIT:C3316]
 xref: DOID:8616 {source="MONDO:equivalentTo"}
+xref: EFO:1000466 {source="MONDO:equivalentTo"}
 xref: ICD10CM:N48.6 {source="DOID:8616"}
 xref: ICD9:607.85 {source="DOID:8616"}
 xref: MESH:D010411 {source="DOID:8616"}
-xref: NCIT:C3316 {source="MONDO:relatedTo", source="DOID:8616"}
+xref: NCIT:C3316 {source="MONDO:equivalentTo", source="EFO:1000466"}
+xref: NCIT:C3316 {source="MONDO:equivalentTo", source="DOID:8616"}
 xref: OMIM:171000 {source="MONDO:equivalentTo", source="DOID:8616"}
```

Comment:
These 2 x-refs are the same, ie these are duplicated. The sources should be grouped together and we should keep only one of them.

Reply:
updated